### DISCUSSION TITLE
You can create playlist with map that used auto

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#17304] [Ideas] [answered]
I think this is wrong
![image](https://user-images.githubusercontent.com/59089206/158872560-77438a8a-c3e4-42e0-8ca9-96de808b3c2e.png)

**Comment by bdach** [comment]
how have you managed to do this? the mod select overlay in playlists mode isn't showing autoplay for me.

![osu_2022-03-17_20-25-22](https://user-images.githubusercontent.com/20418176/158880454-00790a40-235d-4bfb-8996-f538c89b39ef.jpg)

**Reply by Moltanicaa** [reply]
1. Open the game
2. Go to song select
3. Activate Auto
4. Go to playlists
5. Create playlist

**Comment by bdach** [comment, marked as answer]
#17306
